app/blog/2020-09-22-lit-element-and-lit-html-next-preview.md

```diff
@@ -0,0 +1,57 @@
+
+## Motivation
+
+We've been very happy with the current version of our libraries - they're fast, small, and stable (yay!) - and in some ways we _don't_ have very many pressing needs to make breaking changes. We don't take making breaking changes lightly. But there are some compelling reasons for changes that we think will improve the user-experience of components and applications built with LitElement.
+
+1. Performance. We have found that some of our browser-bug workaround code and customization abstractions prevent optimizations that we would like to do.
+2. Size. That same code, and our extensive public API, costs bytes. We always want to find ways to make the libraries smaller.
+3. Features & API cleanup. Some features are difficult to add in a cost-effective way with the current architecture, or can't really be improved without breaking changes.
+4. Server-side-rendering: lit-html has an extremely flexible and customizable API, and in some ways is more of a template-system construction kit than a single template-system. But this flexibility complicates SSR, which needs to make assumptions about how the server-rendered HTML maps to templates. SSR would only work well with the default, uncustomized lit-html, so limiting customization makes SSR more reliable. Very few developers used the customization APIs anyway.
+
+   We also want to make the directive API SSR compatible by limiting access to the DOM.
+
+## What's changed
+
+These are new major versions, so there are some breaking changes, but we want to minimize disruption to our users as much as possible. We've limited the breaking changes so that they don't affect most users, or only require a mechanical change (like changing an import).
+
+Please see the READMEs ([LitElement](https://github.com/Polymer/lit-html/blob/lit-next/packages/lit-element/README.md), [lit-html]((https://github.com/Polymer/lit-html/blob/lit-next/packages/lit-html/README.md))) and CHANGELOGs ([LitElement](https://github.com/Polymer/lit-html/blob/lit-next/packages/lit-element/CHANGELOG.md), [lit-html]((https://github.com/Polymer/lit-html/blob/lit-next/packages/lit-html/CHANGELOG.md))) for the most detailed list of changes.
+
+The most important changes:
+
+  * Customizing the syntax of lit-html is no longer directly supported. The `templateFactory` and `TemplateProcessor` APIs have been removed.
+  * The public API has been minimized in order to facilitate better minification and future evolution.
+  * The lit-html directive API has changed to be class-based and to persist directive instances. Directives should be easier to write and easier to make SSR compatible.
+  * The LitElement decorators are no-longer exported from the main module - they have to be imported individually or from a new `lit-element/decorators.js` module. This means smaller app sizes for non-decorator-users and opens the door to new decorators implementing the current TC39 JavaScript proposal when those arrive.
```

Comment:
```suggestion
  * The LitElement decorators are no-longer exported from the main module—they have to be imported individually or from a new `lit-element/decorators.js` module. This means smaller app sizes for non-decorator-users and opens the door to new decorators implementing the current TC39 JavaScript proposal when those arrive.
```

em dash

Reply:
oh duh. Just noticed that "no-longer" should just be "no longer"

Reply:
And wallah, I can update a suggestion, can't I?

Reply:
TIL I learned how to type a —!